[OAS] Combine manual and automated OpenAPI documents

lcawl opened the pull request:
## Summary

This PR bundles the generated and manually-maintained OpenAPI documents into a single document.


<!--ONMERGE {"backportTargets":["8.15"]} ONMERGE-->

jloleysens commented:
/ci

jloleysens reviewed (commented):
New tag LGTM! Just started a build that should hopefully update the `bundle*json` files with the new tag.

jloleysens commented:
> I've proposed to add status as the tag for the /api/status operations.

This sounds good to me. There is one other API we may consider tagging with this one called `/api/stats`. Users can information about how Kibana is performing (memory usage, concurrent requests etc). If you agree perhaps we can tag them "System" or "Kibana" endpoints?

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/222536)
* Commit: 0e18dc0bce82098e21e42eaec39f4518c980a118

### Failed CI Steps
* [Checks](https://buildkite.com/elastic/kibana-pull-request/builds/222536#0190ca25-1b41-47d0-825f-886e1d52f225)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/222536","number":222536,"commit":"0e18dc0bce82098e21e42eaec39f4518c980a118"}],"number":222536}
buildkite-pr-comment-->

jloleysens commented:
It is auto-committing, but then failing to push to your branch 🤔 

```
Switched to branch 'oas-bundles'
--
  | [oas-bundles 246e0be603ca] [CI] Auto-commit changed files from 'node scripts/capture_oas_snapshot --include-path /api/status --update'
  | 2 files changed, 16 insertions(+), 4 deletions(-)
  | remote: Invalid username or password.
  | fatal: Authentication failed for 'https://github.com/lcawl/kibana.git/'
```

Perhaps you need to trigger the build with `/ci`?

lcawl commented:
> If you agree perhaps we can tag them "System" or "Kibana" endpoints?

I've updated this PR to use "system" instead of "status", thanks.

> Perhaps you need to trigger the build with `/ci`?

I saw that it was running `node scripts/capture_oas_snapshot --include-path /api/status --update` so I've run that myself and added the changed files to this PR. Hopefully now that it's out of draft mode the check will work

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/222680)
* Commit: 379467407745ab3a7c4256b7ed8173d783574f78

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #222536](https://buildkite.com/elastic/kibana-pull-request/builds/222536) failed 0e18dc0bce82098e21e42eaec39f4518c980a118

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/222680","number":222680,"commit":"379467407745ab3a7c4256b7ed8173d783574f78"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/222536","number":222536,"commit":"0e18dc0bce82098e21e42eaec39f4518c980a118"}],"number":222680}
buildkite-pr-comment-->

jloleysens reviewed (approved):
New tag LGTM!

karenzone reviewed (approved):
LGTM

kibanamachine commented:
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.15|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 188702
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 9.4.3 -->

lcawl commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.15|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/188880">](https://github.com/elastic/kibana/pull/188880)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 8.9.8 -->
